06_create_cluster.sh

```diff
@@ -51,22 +51,15 @@ fi
 
 if [[ ! -z "${ENABLE_METALLB_MODE}" ]]; then
 
-	if [[ -z ${METALLB_IMAGE_BASE} ]]; then
-		export METALLB_IMAGE_BASE=$(\
-			jq -r .references.spec.tags[0].from.name ${OCP_DIR}/release_info.json | sed -e 's/@.*$//g')
-		export METALLB_IMAGE_TAG="metallb"
-	fi
+	pushd metallb
+	./configure_metallb.sh
+	popd
 
-	if [[ ${ENABLE_METALLB_MODE} == "l2" ]]; then
-		pushd metallb
-		./metallb_l2.sh
-		popd
-	elif [[ ${ENABLE_METALLB_MODE} == "bgp" ]]; then
+	if [[ ${ENABLE_METALLB_MODE} == "bgp" ]]; then
```

Comment:
Do we need metallb modes anymore?

Reply:
i wanted to leave in this commit the start frr script as it's unrelated to the deployment method change. maybe i better remove the modes in my second PR
https://github.com/openshift-metal3/dev-scripts/pull/1305